core/domain/exp_services_test.py

```diff
@@ -2743,3 +2743,51 @@ def test_get_exploration_and_exploration_rights_object(self):
                 'fake_id'))
         self.assertIsNone(exp)
         self.assertIsNone(exp_rights)
+
+
+class ExplorationStateIDMappingTests(test_utils.GenericTestBase):
```

Comment:
Tests for reverting?

Reply:
There's one in exp_domain_test. Should I move it here?

Reply:
Oh -- I was thinking that exp_services_test should have some test for 
create_and_save_state_id_mapping_model_for_reverted_exploration(). Basically the stuff defined in exp_domain should be tested in exp_domain_test, and the stuff defined in exp_services should be tested in exp_services_test.